src/main/java/com/facebook/openwifirrm/ucentral/prov/models/InventoryConfigApplyResult.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.prov.models;
+
+import java.util.List;
+
+public class InventoryConfigApplyResult {
+	public String appliedConfiguration;
+	public List<String> errors;
+	public List<String> warnings;
+	public Integer errorCode;
```

Comment:
any reason you're using boxed `Integer` instead of primitive `int`? (is the parameter not always filled?)